### PR TITLE
Feature/152 refactor class storage

### PR DESCRIPTION
**Pull request by lukefisklennon**

### CODE REVIEW

**Comment by lukefisklennon** [comment]
Also refactored class times as discussed in the last meeting. Days and hours/minutes are now stored as numbers instead of strings.

**Comment by lukefisklennon** [comment]
Note: time data sent by the server is still sent as strings, so initial conversion has to be done on the client. (We may want to change this?)

**Comment by maxwowo** [comment]
> Note: time data sent by the server is still represented in string form, so initial conversion to numerical form has to be done on the client. (We may want to change this?)

Yep that's ok, we can either refactor server class times later on after either Shanil, Callum or Michael have done their current issues, or I'm happy for you to work on it after you finish the move inventory to sidebar issue. Depends on if you want a break from frontend to play around with our server :)

File: client/src/App.tsx [hunk 1]
```diff
@@ -77,31 +77,26 @@ const App: FunctionComponent = () => {
...
     });
   };
 
-  const handleRemoveClass = (activityId: string) => {
-    setSelectedClassIds((prevSelectedClassIds) => {
-      const newSelectedClassIds = prevSelectedClassIds.filter(
-        (id) => !id.startsWith(activityId),
-      );
-      return newSelectedClassIds;
+  const handleRemoveClass = (classId: string) => {
```
[the remark this answers is not in the document]
Comment (lukefisklennon) [reply client/src/App.tsx]: Done 👍

**Pull request merged**